### PR TITLE
add docker support

### PR DESCRIPTION
**Pull request by jorkzijlstra**
Docker containers are widely used for hosting. This add support to register a docker image so that you can host this anywhere besides Heroku.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,44 @@ http://localhost:9000/?branch={branchName}&accessToken={accessToken}&owner={Owne
...
+  restart: always
+  environment:
+    - JAVA_OPTS=-Dconfig.file=/configs/environment.conf
+  volumes:
+    - ./:/configs
+    - /tmp/resource-api:/logs
+  ports:
+    - "{exposed port}:9000"
+```
+Make sure that the /config directory is available as an volume and that it contains the environment.conf
```
Comment (EECOLOR) [inline README.md]: as **a** volume

File: build.sbt [hunk 2]
```diff
@@ -1,15 +1,85 @@
...
+  dockerExposedPorts in docker := Seq(9000, 9443),
+
+  buildOptions in docker := BuildOptions(cache = false),
+
+  imageNames in docker := dockerImageNames.value,
+
+  dockerImageCreationTask := docker.value
 )
+
+lazy val overrideDockerImageNames = settingKey[Option[Seq[ImageName]]]("get the docker image name")
```
Comment (EECOLOR) [inline build.sbt:74]: Does this really need to be a `Seq`?

**Review by EECOLOR — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jorkzijlstra) [reply README.md]: fixed the type

File: build.sbt (hunk 2 above)
Comment (jorkzijlstra) [reply build.sbt:74]: Yes it does. usually you want to push 2 tags. One with the 'latest' tag and one with the 'version' tag

Comment (EECOLOR) [reply build.sbt:74]: :+1:

**Pull request merged**